mkdocs/utils.py

```diff
@@ -299,10 +300,23 @@ def convert_markdown(markdown_source, extensions=None, extension_configs=None):
     return (html_content, table_of_contents, meta)
 
 
+def get_themes():
+    """Return a dict of theme names and their locations"""
+
+    themes = {}
+
+    for theme in iter_entry_points(group='mkdocs.themes', name=None):
+        theme = theme.load()
+        location = os.path.dirname(theme.__file__)
```

Comment:
I like this. The theme author does not need to write any Python code. In sphinx, the theme author needs to write some code in the `__init__.py` file which returns the location. This is much better. Just include the empty file at the root of the theme, and it becomes available.